package.json

```diff
@@ -126,6 +126,9 @@
     ],
     "!(*sass).md": [
       "prettier --write"
+    ],
+    "**/*.ts": [
+      "eslint"
```

Comment:
Thanks for catching this! Instead of adding this here, I think we should amend the earlier definition for `.js` files on line 119.

```suggestion
```

```diff
-    "**/*.js": [
+    "**/*.{js,jsx,ts,tsx}": [
      "prettier --write",
      "eslint"
    ],
```